test/testother.cpp

```diff
@@ -4799,6 +4799,14 @@ class TestOther : public TestFixture {
             "}");
         ASSERT_EQUALS("[test.cpp:2]: (style) Checking if unsigned expression 'x' is less than zero.\n", errout.str());
 
+        check(
```

Comment:
I don't know why we have written only "check(" on the first line. I like condensed test cases so I can get as much overview as possible when looking at the test code. I don't want that we change the pattern now in this pull request. But if you later want to do that, feel free to convert all the test cases in this file.

Reply:
Agreed, I just tried sticking with the style in the file. I can take a look later. Would it make sense to shorten the test cases further so that
```
bool f(unsigned x) {
    if (x < 0)
        return true;
    return false;
}
```
is replaced with
```
void f(unsigned x) {
    if (x < 0) {}
}
```
(there are quite a few test cases like that and the return statements don't really add anything to the test case)

Reply:
> Agreed, I just tried sticking with the style in the file. 

Yes, that is better than mixing different styles.

> Would it make sense to shorten the test cases further so that

Yes that would be great.